src/collar/oc_dialog.lsl

```diff
@@ -161,13 +160,13 @@ list g_lColors = [
             "Midnight",<0.00000, 0.10588, 0.21176>
 ];
 
-Dialog(key kRecipient, string sPrompt, list lMenuItems, list lUtilityButtons, integer iPage, key kID, integer iWithNums, integer iAuth,string extraInfo, integer iSorted)
+Dialog(key kRecipient, string sPrompt, list lMenuItems, list lUtilityButtons, integer iPage, key kID, integer iWithNums, integer iAuth, string extraInfo, integer iSorted)
 {
     //calculate page start and end
     integer iNumitems = llGetListLength(lMenuItems);
     integer iStart = 0;
     integer iMyPageSize = g_iPagesize - llGetListLength(lUtilityButtons);
-    if (g_iSelectAviMenu) { //we have to reduce buttons due to text length limitations we reach with URI
+    if (extraInfo == "avimenu") { //we have to reduce buttons due to text length limitations we reach with URI

```

Comment:
If you want a menu mode, why not just add an argument for that to the Dialog function?  That would seem to be the clearest way to express what's going on.  The name "extrainfo" seems harder for people to understand how it's used.  Even just changing the name from "extrainfo" to "menuMode" would be OK I think.

Reply:
I suppose in the future extrainfo can be used not only to change dialog mode

Reply:
particles - #530
dialog - #531